client/post-editor/media-modal/detail/detail-file-info.jsx

```diff
@@ -99,6 +100,51 @@ class EditorMediaModalDetailFileInfo extends React.Component {
 		);
 	};
 
+	renderUploadedTo = () => {
+		const parentTitle = this.getItemValue( 'parent_title' );
+		const parentLink = this.getItemValue( 'parent_link' );
+
+		if ( ! parentTitle || ! parentLink ) {
+			return;
+		}
+
+		return (
+			<tr>
+				<th>{ this.props.translate( 'Uploaded to' ) }</th>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **132** times:
`translate( 'Uploaded to', { context: 'column name'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/54691)

Reply:
Looks like it's worth updating this translation to include the `context`, as it's also a column name.